README.md

```diff
@@ -606,27 +740,26 @@ extension Subprocess.Execution {
 
 ```swift
 #if canImport(Glibc) || canImport(Darwin)
-extension Subprocses {
-    /// A step in the graceful shutdown teardown sequence.
-    /// It consists of a signal to send to the child process and the
-    /// duration allowed for the child process to exit before proceeding
-    /// to the next step.
-    @available(FoundationPreview 0.4, *)
-    @available(iOS, unavailable)
-    @available(tvOS, unavailable)
-    @available(watchOS, unavailable)
-    public struct TeardownStep: Sendable, Hashable {
-        /// Sends `signal` to the process and allows `allowedDurationToExit`
-        /// for the process to exit before proceeding to the next step.
-        /// The final step in the sequence will always send a `.kill` signal.
-        public static func sendSignal(
-            _ signal: Signal,
-            allowedDurationToExit: Duration
-        ) -> Self
-    }
+/// A step in the graceful shutdown teardown sequence.
+/// It consists of a signal to send to the child process and the
+/// duration allowed for the child process to exit before proceeding
+/// to the next step.
+@available(FoundationPreview 0.4, *)
```

Comment:
I think this availability should be updated to match the rest, whatever we decide.